plutus-metatheory/src/Algorithmic/Reduction.lagda

```diff
@@ -14,10 +15,11 @@ open import Data.Integer using (_<?_;_+_;_-_;∣_∣;_≤?_;_≟_) renaming (_*_
 open import Relation.Nullary
 open import Relation.Nullary.Decidable
 open import Data.Unit hiding (_≤_; _≤?_; _≟_)
-open import Data.List hiding ([_]; take; drop)
+open import Data.List hiding (take; drop; concat) renaming ([_] to [_]ₗ)
```

Comment:
this unicode character shows up as a box in my browser, so it's probably replaced with a more common one.